lib/win32/certstore/mixin/unicode.rb

```diff
@@ -0,0 +1,49 @@
+#
+# Author:: Vasundhara Jagdale (<vasundhara.jagdale@msystechnologies.com>)
```

Comment:
Include the original Authors..maybe?

Reply:
Yes probably I should add those name my bad.